kolibri/core/content/models.py

```diff
@@ -457,7 +457,7 @@ def build_for_user(cls, user):
         :return: A ContentRequest
         :rtype: ContentRequest
         """
-        return ContentRequest(
+        return cls(
```

Comment:
This seems like an important change! @marcellamaki do you have a similar change in your PR?

Reply:
Yeah I mentioned this on Slack